books/RayTracingTheNextWeek.html

```diff
@@ -957,6 +957,71 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     [Listing [hit-record-uv]: <kbd>[hittable.h]</kbd> Adding U,V coordinates to the `hit_record`]
 
+We will also need to compute $(u,v)$ texture coordinates for hittables.
+
+Texture Coordinates for Spheres
```

Comment:
Two blank lines before H2 headers.

Reply:
I think I'm misunderstanding--I'm seeing blank lines 961 and 962 before line 963: "Texture Coordinates for Spheres". How should this look?